dismorfo/ultraviolet-nightwatch
# Invenio84 file limits size

[Invenio84 Ticket](https://jira.nyu.edu/browse/INVENIO-84)

This PR adds the following:
- tests for the invenio84 ticket - file size/qty warning

To test this PR
- check out branch [invenio84-file-limits-msg-overrides](https://github.com/nyudlts/ultraviolet/tree/invenio84-file-limits-msg-override) in the ultraviolet repository
- startup UV: 
  - `invenio-cli services start`
  - `invenio-cli assets build`
  - `invenio-cli run`
- run the tests:
  - `yarn test ./nightwatch/tests/features/INVENIO-84.js`
  
NOTE: this should work even if we implement the changes with something other than react-overrides.
Thank you for your feedback.
